dagre-d3/dagre-d3.d.ts

```diff
@@ -23,6 +24,8 @@ declare namespace Dagre {
     }
 
     interface Render {
+        // see http://cpettitt.github.io/project/dagre-d3/latest/demo/user-defined.html for example usage
```

Comment:
I don't think this comment is necessary? If you feel strongly about documenting, linking to source might be more useful? https://github.com/cpettitt/dagre-d3/blob/master/lib/arrows.js